chill-java/src/main/java/com/twitter/chill/java/Java8ClosureRegistrar.java

```diff
@@ -0,0 +1,31 @@
+package com.twitter.chill.java;
+
+import com.esotericsoftware.kryo.Kryo;
+import com.twitter.chill.IKryoRegistrar;
+
+/**
+ * Enables Java 8 lambda serialization if running on Java 8; no-op otherwise.
+ */
+public class Java8ClosureRegistrar implements IKryoRegistrar {
+
+    private static boolean checkJava8() {
+        try {
+            Class.forName("java.lang.invoke.SerializedLambda");
+            return true;
+        } catch (ClassNotFoundException e) {
+            // Not running on Java 8.
+            return false;
+        }
+    }
+    private static boolean onJava8Field = checkJava8();
+    public static boolean areOnJava8() {
```

Comment:
I noticed this change wasn't ported to `develop` (see https://github.com/twitter/chill/blob/0919984ec3aeb320ff522911c726425e9f18ea41/chill-java/src/main/java/com/twitter/chill/java/Java8ClosureRegistrar.java#L10 for example). Was it intentional?